src/e2e/puppeteer/helpers/simulateDragAndDrop.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from 'puppeteer'
+
+/** Simulates Drag and Drop functionality in Puppeteer. */
+const dragAndDrop = async (page: Page, selectorDrag: string) => {
+  await page.evaluate(() => {
+    const thoughts = document.querySelectorAll('[aria-label="thought"]')
+    thoughts.forEach((thought, index) => {
+      // Assign an ID
+      thought.id = `thought-${index}`
+    })
+  })
```

Comment:
Let's use the existing [getEditable](https://github.com/cybersemics/em/blob/b20fd295186c0b323c036c70bfa635e9da4a58c5/src/e2e/puppeteer/helpers/getEditable.ts) helper for querying a thought by value.

Reply:
I think getEditable returns the first matching element, we need to get all the thoughts so we might need to create another helper for querying all matching elements, maybe?

Reply:
Querying thoughts by name will make the tests more readable. We will need a way to query thoughts that share a value, but I don't particularly see the need to query all thoughts in a single test.

Reply:
I was using `querySelectorAll` to get the thoughts and add an id at runtime, so any thought could be drag and dropped by using the id. I couldn't find anything else to query the thought with, does the `getEditable` helper use the "value" of the thought?

Reply:
`getEditable` uses XPath, as you can see here: https://github.com/cybersemics/em/blob/b20fd295186c0b323c036c70bfa635e9da4a58c5/src/e2e/puppeteer/helpers/getEditable.ts

Reply:
I understand, we might need to paste thoughts with specific values and then drag and drop, then we wont need to query all thoughts,